datalad/crawler/nodes/tests/test_annex.py

```diff
@@ -24,6 +24,21 @@
 from ....consts import CRAWLER_META_CONFIG_PATH, DATALAD_SPECIAL_REMOTE, ARCHIVES_SPECIAL_REMOTE
 from ....support.stats import ActivityStats
 from ....support.annexrepo import AnnexRepo
+from os import listdir
+
+
+@with_tempfile(mkdir=True)
+def test_Annexificator(outdir):
```

Comment:
call it so it is descriptive of the problem... `test_Annexificator_no_git_if_dirty`